app/views/contacts/show.html.erb

```diff
@@ -5,100 +5,87 @@
-          <%= @contact.email_address %>
-        </div>
-        <hr/>
-        <div class="row font24">
-          <%= @contact.birth_date %>
-        </div>
-        <hr/>
-        <div class="row font24">
-          <%= link_to @ecclesium.name, ecclesium_path(@ecclesium) %>
-        </div>
-        <hr/>
-        <div class="row font24">
-          <%= @contact.postal_address %>
-        </div>
-        <hr/>
-        <div class="row font24">
-          <%= @contact.longitude %>, <%= @contact.latitude %>
-        </div>
-        <hr/>
-        <div class="row font24">
-          <%= @contact.notes  %>
+      <div class="row">
+        <div class="col-12">
+          <div class="list-group list-group-flush">
+            <div class="row list-group-item d-flex">
+              <div class="col-3 text-muted text-right">
+                Phone:
+              </div>
+              <div class="col-9">
+                <%= @contact.phone_number %>
+              </div>
+            </div>
+            <div class="row list-group-item d-flex">
+              <div class="col-3 text-muted text-right">
+                Email:
+              </div>
+              <div class="col-9">
+                <%= @contact.email_address %>
+              </div>
+            </div>
+            <div class="row list-group-item d-flex">
+              <div class="col-3 text-muted text-right">
+                Birthday:
+              </div>
+              <div class="col-9">
+                <%= @contact.birth_date %>
+              </div>
+            </div>
+            <div class="row list-group-item d-flex">
+              <div class="col-3 text-muted text-right">
+                Ecclesia:
+              </div>
+              <div class="col-9">
+                <%= link_to @ecclesium.name, ecclesium_path(@ecclesium) %>
+              </div>
+            </div>
+            <div class="row list-group-item d-flex">
+              <div class="col-3 text-muted text-right">
+                Address:
+              </div>
+              <div class="col-9">
+                <%= @contact.postal_address %>
+              </div>
+            </div>
+            <div class="row list-group-item d-flex">
+              <div class="col-3 text-muted text-right">
+                Coordinates:
+              </div>
+              <div class="col-9">
+                <%= @contact.latitude %>, <%= @contact.longitude %>
+              </div>
+            </div>
+            <div class="row list-group-item d-flex">
+              <div class="col-3 text-muted text-right">
+                Notes:
+              </div>
+              <div class="col-9">
+                <%= @contact.notes %>
+              </div>
+            </div>
+          </div>
         </div>
-
       </div>
-</div>
     </div>
 
-  </div>
-  <div class="col-5">
-    <div id="contactMap">
-
+    <div class="col-5 mt-4">
+      <div id="contact-map"></div>
     </div>
+
   </div>
 </div>
-</div>
```

Comment:
Notice the structure of the page: There's always a row, and then that row has `n` number of columns that add up to 12 (e.g. possibilities include 1 column of `col-12`, or two columns of `col-4` and `col-8`, or three columns of `col-4`, `col-4`, `col-4`, etc). Look at the final product I laid out here to get an idea of how bootstrap utilizes it's grid. 

Also, this will be very helpful: https://getbootstrap.com/docs/4.3/layout/grid/